core/src/main/java/de/valtech/aecu/core/listeners/AecuMigrationTrigger.java

```diff
@@ -0,0 +1,188 @@
+package de.valtech.aecu.core.listeners;
```

Comment:
This class and the commented tests can be removed. We should focus on one way to execute scripts.